packages/orchestrator-ui-components/src/contexts/PolicyContext.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC, ReactNode, createContext, useCallback } from 'react';
+
+export type Policy = {
+    isAllowed: (routerPath: string, resource?: string) => boolean;
+};
+
+export const PolicyContext = createContext<Policy>({ isAllowed: () => true });
+
+export type PolicyProviderProps = {
+    initialIsAllowed: (routerPath: string, resource?: string) => boolean;
+    children: ReactNode;
+};
+
+export const PolicyContextProvider: FC<PolicyProviderProps> = ({
+    initialIsAllowed,
+    children,
+}) => {
+    const isAllowed = useCallback(initialIsAllowed, [initialIsAllowed]);
```

Comment:
What does the useCallback do here?

Reply:
It does nothing, made the project a bit cleaner